proversity_reports_script/report_backend/activity_completion_report.py

```diff
@@ -42,13 +44,14 @@ def json_report_to_csv(self, json_report_data):
                 continue
 
             self.create_csv_file(
-                course,
-                csv_data,
-                csv_data[0].keys(),
+                file_name=course,
+                body_dict=csv_data,
+                course_id=course,
+                headers=csv_data[0].keys(),
```

Comment:
Are we 100% sure that we csv_data[0] holds all the required header keys?

Reply:
All elements in csv_data are dictionaries where the keys are the csv headers so I think we can be sure of this. Please confirm @Squirrel18 At minimum the 'First Name', 'Last Name' and 'Email' will be there.

Reply:
They will be there, if there is at least one record in the report data, otherwise it will fail, because it will be an empty dict, but if there is no record in the report data, it will never enter here. @cocococosti @diegomillan